lib/tailwindcss/commands.rb

```diff
@@ -34,15 +34,15 @@ def executable(exe_path: DEFAULT_DIR)
             MESSAGE
           end
         else
-          if Tailwindcss::Upstream::NATIVE_PLATFORMS.keys.none? { |p| Gem::Platform.match(Gem::Platform.new(p)) }
+          if Tailwindcss::Upstream::NATIVE_PLATFORMS.keys.none? { |p| Gem::Platform.match_gem?(Gem::Platform.new(p), nil) }
```

Comment:
The second argument here should be the gem name, `"tailwindcss-rails"`.

Although this argument isn't used in the CRuby implementation, other implementations (specifically TruffleRuby) reserve the right to re-implement this method with special cases for specific gems. If you'd like to learn more about the history behind this curious design decision, it's all at https://github.com/rubygems/rubygems/pull/3817